projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -103,49 +103,50 @@ describe('ng-add schematics', () => {
     expect(pkgJsonData.dependencies['hammerjs']).toBeTruthy();
   });
 
-  it('should add hammer.js to the main.ts file', async () => {
+  it('should NOT (no longer) add hammer.js to the main.ts file', async () => {
```

Comment:
```suggestion
  it('should NOT add hammer.js to the main.ts file', async () => {
```
Drop the "no longer" thing, there's enough history to trace this has changed, plus "should not no longer do smth" really doesn't work :)